CMakeLists.txt

```diff
@@ -302,7 +302,6 @@ zephyr_compile_options($<$<COMPILE_LANGUAGE:CXX>:$<TARGET_PROPERTY:compiler-cpp,
 include(cmake/extra_flags.cmake)
 
 zephyr_cc_option(-fno-asynchronous-unwind-tables)
-zephyr_cc_option(-fno-pie)
```

Comment:
Why do you want to remove this?

Reply:
Isn't it now added in ld/target_base.cmake anyway? Did I misunderstand how that `.cmake` file gets included?